crates/edr_opt/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "edr_opt"
```

Comment:
It took me a moment to parse the name, I'd prefer `edr_optimism` if possible

Reply:
Addressed by https://github.com/NomicFoundation/edr/pull/534/commits/597c4b860e38635ff44cb9904fec13dff2ae7ad5